ibm/provider/provider.go

```diff
@@ -522,22 +522,26 @@ func Provider() *schema.Provider {
 			"ibm_is_virtual_network_interface_ip":           vpc.DataSourceIBMIsVirtualNetworkInterfaceIP(),
 			"ibm_is_virtual_network_interface_ips":          vpc.DataSourceIBMIsVirtualNetworkInterfaceIPs(),
 
-			"ibm_is_share_mount_target":              vpc.DataSourceIBMIsShareTarget(),
-			"ibm_is_share_mount_targets":             vpc.DataSourceIBMIsShareTargets(),
-			"ibm_is_volume":                          vpc.DataSourceIBMISVolume(),
-			"ibm_is_volumes":                         vpc.DataSourceIBMIsVolumes(),
-			"ibm_is_volume_profile":                  vpc.DataSourceIBMISVolumeProfile(),
-			"ibm_is_volume_profiles":                 vpc.DataSourceIBMISVolumeProfiles(),
-			"ibm_is_vpc":                             vpc.DataSourceIBMISVPC(),
-			"ibm_is_vpc_dns_resolution_binding":      vpc.DataSourceIBMIsVPCDnsResolutionBinding(),
-			"ibm_is_vpc_dns_resolution_bindings":     vpc.DataSourceIBMIsVPCDnsResolutionBindings(),
-			"ibm_is_vpcs":                            vpc.DataSourceIBMISVPCs(),
-			"ibm_is_vpn_gateway":                     vpc.DataSourceIBMISVPNGateway(),
-			"ibm_is_vpn_gateways":                    vpc.DataSourceIBMISVPNGateways(),
-			"ibm_is_vpc_address_prefixes":            vpc.DataSourceIbmIsVpcAddressPrefixes(),
-			"ibm_is_vpc_address_prefix":              vpc.DataSourceIBMIsVPCAddressPrefix(),
-			"ibm_is_vpn_gateway_connection":          vpc.DataSourceIBMISVPNGatewayConnection(),
-			"ibm_is_vpn_gateway_connections":         vpc.DataSourceIBMISVPNGatewayConnections(),
+			"ibm_is_share_mount_target":          vpc.DataSourceIBMIsShareTarget(),
+			"ibm_is_share_mount_targets":         vpc.DataSourceIBMIsShareTargets(),
+			"ibm_is_volume":                      vpc.DataSourceIBMISVolume(),
+			"ibm_is_volumes":                     vpc.DataSourceIBMIsVolumes(),
+			"ibm_is_volume_profile":              vpc.DataSourceIBMISVolumeProfile(),
+			"ibm_is_volume_profiles":             vpc.DataSourceIBMISVolumeProfiles(),
+			"ibm_is_vpc":                         vpc.DataSourceIBMISVPC(),
+			"ibm_is_vpc_dns_resolution_binding":  vpc.DataSourceIBMIsVPCDnsResolutionBinding(),
+			"ibm_is_vpc_dns_resolution_bindings": vpc.DataSourceIBMIsVPCDnsResolutionBindings(),
+			"ibm_is_vpcs":                        vpc.DataSourceIBMISVPCs(),
+			"ibm_is_vpn_gateway":                 vpc.DataSourceIBMISVPNGateway(),
+			"ibm_is_vpn_gateways":                vpc.DataSourceIBMISVPNGateways(),
+			"ibm_is_vpc_address_prefixes":        vpc.DataSourceIbmIsVpcAddressPrefixes(),
+			"ibm_is_vpc_address_prefix":          vpc.DataSourceIBMIsVPCAddressPrefix(),
+			"ibm_is_vpn_gateway_connection":      vpc.DataSourceIBMISVPNGatewayConnection(),
+			"ibm_is_vpn_gateway_connections":     vpc.DataSourceIBMISVPNGatewayConnections(),
+
+			"ibm_is_vpn_gateway_connection_local_cidrs": vpc.DataSourceIBMIsVPNGatewayConnectionLocalCidrs(),
+			"ibm_is_vpn_gateway_connection_peer_cidrs":  vpc.DataSourceIBMIsVPNGatewayConnectionPeerCidrs(),
+
```

Comment:
Should this change be in this PR?
`			"ibm_is_vpn_gateway_connection_local_cidrs": vpc.DataSourceIBMIsVPNGatewayConnectionLocalCidrs(),
			"ibm_is_vpn_gateway_connection_peer_cidrs":  vpc.DataSourceIBMIsVPNGatewayConnectionPeerCidrs(),`